spec/Section 6 -- Execution.md

```diff
@@ -18,9 +18,9 @@ A GraphQL service generates a response from a request via execution.
 - {extensions} (optional): A map reserved for implementation-specific additional
   information.
 
-Given this information, the result of {ExecuteRequest(schema, document,
-operationName, variableValues, initialValue)} produces the response, to be
-formatted according to the Response section below.
+Given this information, the result of {Request(schema, document, operationName,
```

Comment:
Lets think of how we can keep the algorithm names with action verbs. Ideally this has an "execute" or "process" or something here.

Considering the prose above describes this as "request via execution" then "execute request" does seem like what a reader would expect